README.md

```diff
@@ -0,0 +1,39 @@
+# Profiles
+
+A collection of security profiles for containers, including AppArmor and seccomp profiles used by Moby.
+
+## Overview
+
+This repository contains:
+
+- **AppArmor profiles**: Linux kernel security module profiles for mandatory access control
+- **seccomp profiles**: Secure computing mode profiles for syscall filtering
+
+## Components
+
+### AppArmor (`/apparmor`)
+
+The AppArmor package provides functionality for generating and managing AppArmor profiles for containers.
+
+### seccomp (`/seccomp`)
+
+The seccomp package provides functionality for generating and managing seccomp profiles for syscall filtering in containers.
+
+## Usage
+
+Each package can be imported and used independently:
+
+```go
+import (
+    "github.com/moby/profiles/apparmor"
+    "github.com/moby/profiles/seccomp"
+)
+```
+
+## License
+
+This project is licensed under the Apache License 2.0. See [LICENSE](LICENSE) for details.
```

Comment:
We still need to add proper license headers to our code (also other projects); I like the approach some projects took with just the SPDX headers and a 1-line copyright (can be multiple), but I need to dig up some research I did on correctness; https://github.com/google/oss-rebuild/blob/6f57c474a8faf3012204792af8ef5d8b6fae2fd1/cmd/proxy/main.go#L2